src/Bot/Links.hs

```diff
@@ -145,5 +145,5 @@ linkFilter reaction =
              sender
              [qms|Links are not allowed for untrusted users.
                   Subscribe to gain the trust instantly:
-                  https://twitch.tv/tsoding/subscribe|]
+                  https://twitch.tv/{channel}/subscribe|]
```

Comment:
Instead of tediously forwarding the channel name, you can take the channel name from the `senderChannel` field of the `Sender` record above. :wink:

Reply:
I found a hammer and saw nails. You're right, I didn't realize I could get it out of the reaction at the time. Should be fixed in next commit